pkg/appmanager/resourceConfig.go

```diff
@@ -1679,13 +1659,9 @@ func (appMgr *Manager) createRSConfigFromRoute(
 	var appRootRules []*Rule
 	if appRoot, ok := route.ObjectMeta.Annotations[f5VsAppRootAnnotation]; ok {
 		appRootMap := parseAppRootURLRewriteAnnotations(appRoot)
-		if len(appRootMap) > 1 {
-			log.Warningf("App root annotation: %s does not support multiple values for routes, not processing", appRoot)
```

Comment:
Same comment above about this warning not being moved somewhere else.

Reply:
This moved to validateResources line 265